packages/devtools_app/lib/src/trace_event.dart

```diff
@@ -140,6 +137,18 @@ class TraceEventWrapper implements Comparable<TraceEventWrapper> {
   }
 }
 
+String generateAsyncUID({
+  @required String id,
+  @required String category,
+  String scope,
+}) {
+  if (scope == null) {
```

Comment:
nit:
```dart
[category, if (scope != null) scope, id].join(':')
```
is probably a bit slower but easier to read.

Reply:
done